utils/gyb_syntax_support/AvailabilityNodes.py

```diff
@@ -0,0 +1,66 @@
+from Child import Child
+from Node import Node  # noqa: I201
+
+AVAILABILITY_NODES = [
+    # availability-spec-list -> availability-entry availability-spec-list?
+    Node('AvailabilitySpecList', kind='SyntaxCollection', 
```

Comment:
can we add some doc-comments for these new nodes, i.e. `description=''`. The description will populated as doc-comments in the generated code so that the end users will see.